2017-10-07 19:39:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: テーブル "PERSONALTABEL" が見つかりません
Table "PERSONALTABEL" not found; SQL statement:
DELETE FROM PERSONALTABEL WHERE ID=? [42102-196]
2017-10-07 19:39:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: テーブル "PERSONALTABEL" が見つかりません
Table "PERSONALTABEL" not found; SQL statement:
DELETE FROM PERSONALTABEL WHERE ID=? [42102-196]
2017-10-07 19:40:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: テーブル "PERSONALTABEL" が見つかりません
Table "PERSONALTABEL" not found; SQL statement:
DELETE FROM PERSONALTABEL WHERE ID=? [42102-196]
2017-10-07 19:41:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: テーブル "PERSONALTABEL" が見つかりません
Table "PERSONALTABEL" not found; SQL statement:
DELETE FROM PERSONALTABEL WHERE ID=? [42102-196]
2017-10-07 20:06:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "VARCHARA"
Unknown data type: "VARCHARA"; SQL statement:
ALTER TABLE MESSAGETABLE ADD ID VARCHARA(10) AFTER MESSAGEID [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5903)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5615)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:05:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show CALENDARTABLE [42001-196]
2017-10-10 14:07:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: テーブル "SHOPPINGID" が見つかりません
Table "SHOPPINGID" not found; SQL statement:
alter table SHOPPINGID  alter column SHOPPINGID int(10) [42102-196]
2017-10-10 14:14:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "CONTENS" が見つかりません
Column "CONTENS" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENS,DONE) values("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.getColumn(Table.java:680)
	at org.h2.command.Parser.parseColumn(Parser.java:887)
	at org.h2.command.Parser.parseColumnList(Parser.java:871)
	at org.h2.command.Parser.parseInsert(Parser.java:1071)
	at org.h2.command.Parser.parsePrepared(Parser.java:417)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:15:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:16:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:18:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:18:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:20:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:28:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:29:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:30:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:32:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:33:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0001" が見つかりません
Column "0001" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0001","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:33:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "0002" が見つかりません
Column "0002" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ("0002","2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:34:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "2017/01/01" が見つかりません
Column "2017/01/01" not found; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ('0001',"2017/01/01","test",true) [42122-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Insert.prepare(Insert.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:263)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 14:34:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: "DATE" 定数 "2017/01/01" を解析できません
Cannot parse "DATE" constant "2017/01/01"; SQL statement:
insert into TODOTABLE (GROUPID,DATE,CONTENTS,DONE) values ('0001','2017/01/01','test',true) -- ('0001', '2017/01/01', 'test', TRUE) [22007-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:948)
	at org.h2.table.Column.convert(Column.java:167)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalArgumentException: 2017/01/01
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:313)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 15 more
2017-10-10 14:35:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "INSERT INTO TODOTABLE (TODOID,GROUPID,DATE,CONTENTS,DONE) VALUES (,[*]'0001','2017-01-01','test',TRUE) "; 期待されるステートメント "), DEFAULT, NOT, EXISTS, INTERSECTS, SELECT, FROM, WITH"
Syntax error in SQL statement "INSERT INTO TODOTABLE (TODOID,GROUPID,DATE,CONTENTS,DONE) VALUES (,[*]'0001','2017-01-01','test',TRUE) "; expected "), DEFAULT, NOT, EXISTS, INTERSECTS, SELECT, FROM, WITH"; SQL statement:
insert into TODOTABLE (TODOID,GROUPID,DATE,CONTENTS,DONE) values (,'0001','2017-01-01','test',true) [42001-196]
2017-10-10 14:38:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table status [42001-196]
2017-10-10 14:38:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table status from TODOTABLE [42001-196]
2017-10-10 14:38:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table status from TODOTABLE [42001-196]
2017-10-10 14:38:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table [42001-196]
2017-10-10 14:39:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "DESCRIBE[*] TODOTABLE "; 期待されるステートメント "DELETE, DROP, DECLARE, DEALLOCATE, {"
Syntax error in SQL statement "DESCRIBE[*] TODOTABLE "; expected "DELETE, DROP, DECLARE, DEALLOCATE, {"; SQL statement:
describe TODOTABLE [42001-196]
2017-10-10 14:40:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show variables [42001-196]
2017-10-10 14:40:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show variables [42001-196]
2017-10-10 14:40:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show variables like '%auto_inc%' from TODOTABLE [42001-196]
2017-10-10 14:40:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show variables from TODOTABLE [42001-196]
2017-10-10 14:40:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show variables from TODOTABLE [42001-196]
2017-10-10 14:40:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show variables from TODOTABLE [42001-196]
2017-10-10 14:41:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table status like 'TODOTABLE ' [42001-196]
2017-10-10 14:41:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table status like 'TODOTABLE' [42001-196]
2017-10-10 14:43:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT * FROM 'TODOTABLE'[*] "; 期待されるステートメント "identifier"
Syntax error in SQL statement "SELECT * FROM 'TODOTABLE'[*] "; expected "identifier"; SQL statement:
select * from 'TODOTABLE' [42001-196]
2017-10-10 14:44:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table status from TODOTABLE [42001-196]
2017-10-10 14:44:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show table status  TODOTABLE [42001-196]
2017-10-10 14:45:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show columuns from TODOTABLE [42001-196]
2017-10-10 14:45:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show columuns from TODOTABLE [42001-196]
2017-10-10 14:45:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show columuns from TODOTABLE [42001-196]
2017-10-10 14:45:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show columuns from TODOTABLE [42001-196]
2017-10-10 14:45:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show columuns from TODOTABLE [42001-196]
2017-10-10 14:45:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SELECT [*]"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"
Syntax error in SQL statement "SELECT [*]"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS"; SQL statement:
show columuns from TODOTABLE [42001-196]
2017-10-10 15:12:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "AUTO_INCREMENT"
Unknown data type: "AUTO_INCREMENT"; SQL statement:
ALTER TABLE TODOTABLE ALTER TODOID AUTO_INCREMENT [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:5865)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5834)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:13:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "AUTO_INCREMENT"
Unknown data type: "AUTO_INCREMENT"; SQL statement:
ALTER TABLE TODOTABLE ALTER TODOID auto_increment [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAlterColumnType(Parser.java:5865)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5834)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:16:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "AUTO_INCREMENT"
Unknown data type: "AUTO_INCREMENT"; SQL statement:
ALTER TABLE TODOTABLE  ADD todoid auto_increment [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5903)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5615)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:16:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列名 "TODOID" が重複しています
Duplicate column name "TODOID"; SQL statement:
ALTER TABLE TODOTABLE  ADD todoid int(10) auto_increment [42121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.table.Table.setColumns(Table.java:428)
	at org.h2.table.TableBase.<init>(TableBase.java:45)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:103)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:109)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:42)
	at org.h2.schema.Schema.createTable(Schema.java:612)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:371)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:267)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:184)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:18:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "("
Unknown data type: "("; SQL statement:
ALTER TABLE TODOTABLE  CHANGE todoid int(10) AUTO_INCREMENT [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5733)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:20:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "ID"
Unknown data type: "ID"; SQL statement:
ALTER TABLE TODOTABLE  CHANGE todoid todoid  id identity [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5733)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:26:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "CREATE TABLE TODOTABLE (
TODOID INT(10) PRIMARY KEY AUTOINCREMENT[*],
GROUPID VARCHAR(10),
DATE DATE,
CONTENTS VARCHAR(100),
DONE BOOLEAN) "; 期待されるステートメント "HASH, AUTO_INCREMENT, NOT, NULL, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE TODOTABLE (
TODOID INT(10) PRIMARY KEY AUTOINCREMENT[*],
GROUPID VARCHAR(10),
DATE DATE,
CONTENTS VARCHAR(100),
DONE BOOLEAN) "; expected "HASH, AUTO_INCREMENT, NOT, NULL, CHECK, REFERENCES, ,, )"; SQL statement:
create table TODOTABLE (
TODOID int(10) PRIMARY KEY AUTOINCREMENT,
GROUPID varchar(10),
DATE date,
CONTENTS varchar(100),
DONE BOOLEAN) [42001-196]
2017-10-10 15:28:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "INSERT INTO ([*]GROUPID) VALUES ('0001') "; 期待されるステートメント "identifier"
Syntax error in SQL statement "INSERT INTO ([*]GROUPID) VALUES ('0001') "; expected "identifier"; SQL statement:
insert into (GROUPID) values ('0001') [42001-196]
2017-10-10 15:30:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: テーブル "VALUES" が見つかりません
Table "VALUES" not found; SQL statement:
insert into values (1,'0001',2017-01-01,'test',true) [42102-196]
2017-10-10 15:30:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: "DATE" 定数 "2015" を解析できません
Cannot parse "DATE" constant "2015"; SQL statement:
insert into TODOTABLE  values (1,'0001',2017-01-01,'test',true) -- (1, '0001', 2015, 'test', TRUE) [22007-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:948)
	at org.h2.table.Column.convert(Column.java:167)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalArgumentException: 2015
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:313)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 15 more
2017-10-10 15:42:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "NOT"
Unknown data type: "NOT"; SQL statement:
ALTER TABLE MESSAGETABLE ADD ID NOT NULL [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5903)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5615)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:42:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "NOTNULL"
Unknown data type: "NOTNULL"; SQL statement:
ALTER TABLE MESSAGETABLE ADD ID NOTNULL [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5903)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5615)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:42:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 不明なデータ型: "NOT_NULL"
Unknown data type: "NOT_NULL"; SQL statement:
ALTER TABLE MESSAGETABLE ADD ID NOT_NULL [50004-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4169)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4012)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:5903)
	at org.h2.command.Parser.parseAlterTable(Parser.java:5615)
	at org.h2.command.Parser.parseAlter(Parser.java:5007)
	at org.h2.command.Parser.parsePrepared(Parser.java:348)
	at org.h2.command.Parser.parse(Parser.java:321)
	at org.h2.command.Parser.parse(Parser.java:293)
	at org.h2.command.Parser.prepareCommand(Parser.java:258)
	at org.h2.engine.Session.prepareLocal(Session.java:578)
	at org.h2.engine.Session.prepareCommand(Session.java:519)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1204)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:176)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:43:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "ALTER TABLE MESSAGETABLE ADD ID VARCHAR(10) NOT_NULL[*] "
Syntax error in SQL statement "ALTER TABLE MESSAGETABLE ADD ID VARCHAR(10) NOT_NULL[*] "; SQL statement:
ALTER TABLE MESSAGETABLE ADD ID varchar(10) NOT_NULL [42000-196]
2017-10-10 15:44:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "ALTER TABLE MESSAGETABLE ALTER COLUMN ID VARCHAR(10) NOT_NULL[*] "
Syntax error in SQL statement "ALTER TABLE MESSAGETABLE ALTER COLUMN ID VARCHAR(10) NOT_NULL[*] "; SQL statement:
ALTER TABLE MESSAGETABLE ALTER COLUMN ID varchar(10) NOT_NULL [42000-196]
2017-10-10 15:44:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "ID" がnull値を含んでいます
Column "ID" contains null values; SQL statement:
ALTER TABLE MESSAGETABLE ALTER COLUMN ID varchar(10) NOT NULL [90081-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAlterColumn.checkNoNullValues(AlterTableAlterColumn.java:528)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:158)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:46:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 列 "ID" がnull値を含んでいます
Column "ID" contains null values; SQL statement:
ALTER TABLE MESSAGETABLE ALTER COLUMN ID varchar(10) NOT NULL [90081-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAlterColumn.checkNoNullValues(AlterTableAlterColumn.java:528)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:158)
	at org.h2.command.CommandContainer.update(CommandContainer.java:101)
	at org.h2.command.Command.executeUpdate(Command.java:260)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:192)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:164)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1380)
	at org.h2.server.web.WebApp.query(WebApp.java:1053)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1015)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1002)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Thread.java:745)
2017-10-10 15:51:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "insert into MESSAGETABLE (GROUPID,DATE,MESSAGE) values (''0001','2017-01-01','test[*]')"
Syntax error in SQL statement "insert into MESSAGETABLE (GROUPID,DATE,MESSAGE) values (''0001','2017-01-01','test[*]')"; SQL statement:
insert into MESSAGETABLE (GROUPID,DATE,MESSAGE) values (''0001','2017-01-01','test') [42000-196]
2017-10-10 15:55:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "DELETE FROM TODOTABLE WEHER TODOID[*] = 2"
Syntax error in SQL statement "DELETE FROM TODOTABLE WEHER TODOID[*] = 2"; SQL statement:
DELETE FROM TODOTABLE WEHER TODOID = 2 [42000-196]
2017-10-10 15:56:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "DELETE FROM TODOTABLE WEHRE TODOID[*] = 2"
Syntax error in SQL statement "DELETE FROM TODOTABLE WEHRE TODOID[*] = 2"; SQL statement:
DELETE FROM TODOTABLE WEHRE TODOID = 2 [42000-196]
2017-10-10 15:56:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"
Syntax error in SQL statement "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"; SQL statement:
DELETE FROM TODOTABLE WEHRE GROUPID = '0001' [42000-196]
2017-10-10 15:57:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"
Syntax error in SQL statement "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"; SQL statement:
DELETE FROM TODOTABLE WEHRE GROUPID = '0001' [42000-196]
2017-10-10 15:57:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"
Syntax error in SQL statement "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"; SQL statement:
DELETE FROM TODOTABLE WEHRE GROUPID = '0001' [42000-196]
2017-10-10 15:58:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"
Syntax error in SQL statement "DELETE FROM TODOTABLE WEHRE GROUPID[*] = '0001'"; SQL statement:
DELETE FROM TODOTABLE WEHRE GROUPID = '0001' [42000-196]
2017-10-10 16:33:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "SHOW COLUMNS PERSONALTABLE[*] "; 期待されるステートメント "FROM"
Syntax error in SQL statement "SHOW COLUMNS PERSONALTABLE[*] "; expected "FROM"; SQL statement:
show columns PERSONALTABLE [42001-196]
2017-10-10 16:33:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "ALTER TBALE[*] CALENDARTABLE ADD NAME VARCHAR(30) "; 期待されるステートメント "TABLE, USER, INDEX, SCHEMA, SEQUENCE, VIEW"
Syntax error in SQL statement "ALTER TBALE[*] CALENDARTABLE ADD NAME VARCHAR(30) "; expected "TABLE, USER, INDEX, SCHEMA, SEQUENCE, VIEW"; SQL statement:
ALTER TBALE CALENDARTABLE ADD NAME varchar(30) [42001-196]
2017-10-10 16:35:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "ALTER TBALE[*] CALENDARTABLE ADD NAME VARCHAR(30) "; 期待されるステートメント "TABLE, USER, INDEX, SCHEMA, SEQUENCE, VIEW"
Syntax error in SQL statement "ALTER TBALE[*] CALENDARTABLE ADD NAME VARCHAR(30) "; expected "TABLE, USER, INDEX, SCHEMA, SEQUENCE, VIEW"; SQL statement:
ALTER TBALE CALENDARTABLE ADD NAME varchar(30) [42001-196]
2017-10-10 16:35:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "ALTER TBALE[*] CALENDARTABLE ADD NAME VARCHAR(30) "; 期待されるステートメント "TABLE, USER, INDEX, SCHEMA, SEQUENCE, VIEW"
Syntax error in SQL statement "ALTER TBALE[*] CALENDARTABLE ADD NAME VARCHAR(30) "; expected "TABLE, USER, INDEX, SCHEMA, SEQUENCE, VIEW"; SQL statement:
ALTER TBALE CALENDARTABLE ADD NAME varchar(30) [42001-196]
2017-10-12 15:35:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: パラメータ "#2" がセットされていません
Parameter "#2" is not set; SQL statement:
SELECT * FROM PERSONALTABLE WHERE ID=? AND PW=? [90012-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.Parameter.checkSet(Parameter.java:81)
	at org.h2.command.Prepared.checkParameters(Prepared.java:164)
	at org.h2.command.CommandContainer.query(CommandContainer.java:112)
	at org.h2.command.Command.executeQuery(Command.java:201)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:111)
	at dao.PersonalDAO.loginCheck(PersonalDAO.java:219)
	at servlet.DbTest.main(DbTest.java:35)
2017-10-12 15:37:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:37:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:38:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:38:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:40:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:41:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:42:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:42:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:43:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:45:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:45:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 15:46:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-12 21:58:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: 有効なデータがありません
No data is available [2000-196]
2017-10-15 05:51:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "INSERT INTO PERSONALTABLE (GROUPID) VALUES ('test') WHERE[*] ID = 'hoge' "
Syntax error in SQL statement "INSERT INTO PERSONALTABLE (GROUPID) VALUES ('test') WHERE[*] ID = 'hoge' "; SQL statement:
insert into PERSONALTABLE (groupid) values ('test') where id = 'hoge' [42000-196]
2017-10-15 05:52:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "INSERT INTO PERSONALTABLE (GROUPID) VALUES ('test') WHERE[*] ID = 'hoge' "
Syntax error in SQL statement "INSERT INTO PERSONALTABLE (GROUPID) VALUES ('test') WHERE[*] ID = 'hoge' "; SQL statement:
insert into PERSONALTABLE (GROUPID) values ('test') where id = 'hoge' [42000-196]
2017-10-15 05:56:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: SQLステートメントに文法エラーがあります "update PERSONALTABLE  set groupid = '0001' w'here id = 'hoge[*]'"
Syntax error in SQL statement "update PERSONALTABLE  set groupid = '0001' w'here id = 'hoge[*]'"; SQL statement:
update PERSONALTABLE  set groupid = '0001' w'here id = 'hoge' [42000-196]
2017-10-15 05:59:31 database: flush
org.h2.message.DbException: 一般エラー: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at dao.PersonalDAO.findAll(PersonalDAO.java:29)
	at servlet.DbTest.main(DbTest.java:13)
Caused by: org.h2.jdbc.JdbcSQLException: 一般エラー: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 20 more
2017-10-15 05:59:34 database: flush
org.h2.message.DbException: 一般エラー: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.panic(MVStore.java:378)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:361)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2930)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2476)
	at org.h2.engine.Database.open(Database.java:697)
	at org.h2.engine.Database.openDatabase(Database.java:276)
	at org.h2.engine.Database.<init>(Database.java:270)
	at org.h2.engine.Engine.openSession(Engine.java:64)
	at org.h2.engine.Engine.openSession(Engine.java:176)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
	at org.h2.engine.Engine.createSession(Engine.java:137)
	at org.h2.engine.Engine.createSession(Engine.java:27)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:354)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:116)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:100)
	at org.h2.Driver.connect(Driver.java:69)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at dao.PersonalDAO.findAll(PersonalDAO.java:29)
	at servlet.DbTest.main(DbTest.java:13)
Caused by: org.h2.jdbc.JdbcSQLException: 一般エラー: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]" [50000-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 25 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/K/Desktop/work/SD-2017/H2DB/sd-2017db.mv.db [1.4.196/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:765)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:347)
	... 20 more
2017-10-15 06:02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: パラメータ "parameterIndex" に対する値 "1" が不正です
Invalid value "1" for parameter "parameterIndex" [90008-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1444)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:363)
	at dao.PersonalDAO.setGroupId(PersonalDAO.java:211)
	at servlet.DbTest.main(DbTest.java:25)
2017-10-15 06:03:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: パラメータ "parameterIndex" に対する値 "1" が不正です
Invalid value "1" for parameter "parameterIndex" [90008-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1444)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:363)
	at dao.PersonalDAO.setGroupId(PersonalDAO.java:211)
	at servlet.DbTest.main(DbTest.java:25)
2017-10-15 06:04:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: パラメータ "parameterIndex" に対する値 "1" が不正です
Invalid value "1" for parameter "parameterIndex" [90008-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1444)
	at org.h2.jdbc.JdbcPreparedStatement.setString(JdbcPreparedStatement.java:363)
	at dao.PersonalDAO.setGroupId(PersonalDAO.java:211)
	at servlet.DbTest.main(DbTest.java:25)
